x/finality/client/cli/tx.go

```diff
@@ -167,37 +165,3 @@ func NewAddFinalitySigCmd() *cobra.Command {
 
 	return cmd
 }
-
-func NewUnjailFinalityProviderCmd() *cobra.Command {
-	cmd := &cobra.Command{
-		Use:   "unjail-finality-provider [fp_btc_pk]",
```

Comment:
why remove it? do we not have the unjail option @gitferry

Reply:
Oops maybe this is an error due to rebase. Adding it back